src/main/java/com/exactpro/th2/common/schema/filter/strategy/impl/AbstractFilterStrategy.java

```diff
@@ -58,7 +56,7 @@ private boolean checkValues(Map<String, String> messageFields, MultiValuedMap<St
         return fieldFilters.isEmpty() || fieldFilters.keys().stream().anyMatch(fieldName -> {
             String messageValue = messageFields.get(fieldName);
             Collection<FieldFilterConfiguration> filters = fieldFilters.get(fieldName);
-            return !filters.isEmpty() && filters.stream().allMatch(filter -> valueChecker.check(messageValue, filter));
+            return !filters.isEmpty() && filters.stream().allMatch(filter -> FieldValueCheckerKt.checkFieldValue(messageValue, filter));
```

Comment:
> FieldValueCheckerKt

I suggest to remove `Kt` suffix by overriding default name with `@file:JvmName` annotation